pkg/cmd/pipelinerun/describe.go

```diff
@@ -18,14 +18,15 @@ import (
 	"fmt"
 	"io"
 
+	"text/tabwriter"
```

Comment:
was this change intentional? seems unrelated

Reply:
My vs code put it at the top, will fix it.

Reply:
it should respect  whatever we have in `go fmt` and `goimports` isnt it ?